src/atoms/AllerLogo.js

```diff
@@ -0,0 +1,58 @@
+import React, { Fragment } from 'react';
+import styled from 'react-emotion';
+import PropTypes from 'prop-types';
+
+const Svg = styled.svg`
+	& .st0 {
+		fill: ${props => props.fill};
+	}
+`;
+
+/* eslint-disable max-len */
+
+/** Aller Main Logo */
+const AllerLogo = ({ fill }) => (
```

Comment:
is in not better to spread the props over the svg icon?

Reply:
I agree with Sindre, but I'm not passionate about it.